src/FSharp.Data.GraphQL.Server.AppInfrastructure/GraphQLWebsocketMiddleware.fs

```diff
@@ -107,7 +107,7 @@ type GraphQLWebSocketMiddleware<'Root>(next : RequestDelegate, applicationLifeti
     task {
       if not (socket.State = WebSocketState.Open) then
         if logger.IsEnabled(LogLevel.Trace) then
-          logger.LogTrace("Ignoring message to be sent via socket, since its state is not 'Open', but '{state}'" socket.State)
+          logger.LogTrace("Ignoring message to be sent via socket, since its state is not 'Open', but '{state}'", (socket.State.ToString()))
```

Comment:
I suppose it will be converted to a string on output, no?

Reply:
By the way you can do `{state:format}`if there is some

Reply:
Done.